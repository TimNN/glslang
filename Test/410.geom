#version 410 core

void main()
{
    gl_ViewportIndex = 7;
}
