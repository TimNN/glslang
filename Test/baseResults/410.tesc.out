410.tesc
Warning, version 400 is not yet complete; some version-specific features are present, but many are missing.
ERROR: 0:4: 'length' :  array must first be sized by a redeclaration or layout qualifier
ERROR: 1 compilation errors.  No code generated.


vertices = 0
ERROR: node is still EOpNull!
0:8  Function Definition: main( (void)
0:8    Function Parameters: 
0:?   Linker Objects
0:?     'outa' (1-element array of int)
0:?     'patchOut' (patch out 4-component vector of float)


Linked tessellation control stage:

ERROR: Linking tessellation control stage: At least one shader must specify an output layout(vertices=...)

vertices = 0

